src/test/java/com/github/bibsysdev/nva/cristin/person/CristinPersonsHandlerTest.java

```diff
@@ -0,0 +1,227 @@
+import static nva.commons.core.attempt.Try.attempt;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.bibsysdev.nva.cristin.person.model.cristin.CristinPerson;
+import com.github.bibsysdev.nva.cristin.person.model.nva.NvaPerson;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import no.unit.nva.testutils.HandlerRequestBuilder;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.GatewayResponse;
+import nva.commons.core.Environment;
+import nva.commons.core.JsonUtils;
+import nva.commons.core.ioutils.IoUtils;
+import org.apache.commons.codec.Charsets;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+import org.mockito.Mockito;
+
+public class CristinPersonsHandlerTest {
+
+    private static final String LANGUAGE_KEY = "language";
+    private static final String EMPTY_STRING = "";
+    private static final String LANGUAGE_NB = "nb";
+    private static final String LANGUAGE_INVALID = "invalid";
+    private static final String NAME_MATTIAS = "Mattias";
+    private static final String INVALID_JSON = "This is not valid JSON!";
+    private static final String EMPTY_LIST_STRING = "[]";
+    private static final String ALLOW_ALL_ORIGIN = "*";
+    private static final String NVA_ONE_PERSON_JSON_FILE = "nva_one_person.json";
+    private static final String NVA_PERSONS_NON_ENRICHED_JSON_FILE = "nva_persons_non_enriched.json";
+    private static final String NVA_PERSONS_NO_HITS_JSON_FILE = "nva_persons_no_hits.json";
+    private static final String CRISTIN_ONE_PERSON_JSON_FILE = "cristin_one_person.json";
+    private final ObjectMapper objectMapper = JsonUtils.objectMapper;
+    private final Environment environment = new Environment();
+    private CristinApiClient cristinApiClientStub;
+    private Context context;
+    private ByteArrayOutputStream output;
+    private CristinPersonsHandler handler;
+
+    @Test
```

Comment:
add test expecting the context node.

Reply:
Can you be more specific about what you want to test?

Reply:
The expected item should be a Java class and the actual item should be a Java class as well. This will help in finding out  what are the inconsistencies with the resource files when the Java classes will change. You can use the method GatewayResponse::getBodyObject to get the POJO from the gateway reponse

Reply:
I'll create a JIRA task for this as a future improvement to testing, most likely to be done for some common library,